frontend/src/scenes/banner/BannerPageComponents/BannerLeftContent/BannerLeftContent.jsx

```
@@ -1,54 +1,146 @@
+  const handleSetUrl = (event) => {
+    setUrl(event.target.value);
+  };
+
+  const handleSetActionUrl = (event) => {
+    setActionUrl(event.target.value);
+  };
+
+  const handleActionChange = (newAction) => {
+    setButtonAction(newAction);
+  };
+  const handleActionChange = (newAction) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handleActionChange'. Consider to delete it or rename it.

'handleActionChange' is defined here:


(lint/suspicious/noRedeclare)